reference/OpenAPI.json

```diff
@@ -806,38 +946,164 @@
             "type": "string",
             "description": "The merchant’s product/SKU ID (if it is different from the Voucherify product/SKU ID). It is really useful in case of integration between multiple systems. It can be an ID from an eCommerce site, a database or a 3rd party service."
           },
+          "quantity": {
+            "type": "integer",
+            "description": "The quantity of the particular item in the cart."
+          },
           "discount_quantity": {
             "type": "integer",
             "description": "Number of dicounted items."
           },
           "initial_quantity": {
-            "type": "integer"
+            "type": "integer",
+            "description": "A positive integer in the smallest unit quantity representing the total amount of the order; this is the sum of the order items' quantity."
           },
-          "quantity": {
+          "amount": {
             "type": "integer",
-            "description": "The quantity of the particular item in the cart."
+            "description": "The total amount of the order item (price * quantity)."
+          },
+          "discount_amount": {
+            "type": "integer",
+            "description": "\nSum of all order-item-level discounts applied to the order."
+          },
+          "initial_amount": {
+            "type": "integer",
+            "description": "A positive integer in the smallest currency unit (that is, 100 cents for $1.00) representing the total amount of the order; this is the sum of the order items' amounts."
           },
           "price": {
-            "type": "number",
+            "type": "integer",
             "description": "Unit price of an item. Value is multiplied by 100 to precisely represent 2 decimal places. For example, $100 is written as 10000."
           },
+          "product": {
```

Comment:
To będzie jakoś później zmieniane jak rozumiem? Bo patrząc w openApi to jest dużo takich obiektów produktu wiec warto by to było zunifikować.

Reply:
tak, nie widze w tym problemu